lib/presentation/mixin/load_more_contacts_mixin.dart

```diff
@@ -13,12 +13,16 @@ mixin LoadMoreContactsMixin {
   Set<PresentationContact> oldContactsList = {};
   bool isLoadMore = true;
 
+  bool allowLoadMore = true;
+
   void listenForScrollChanged({required FetchContactsController fetchContactsController}) {
     scrollController.addListener(() {
-      if (isLoadMoreAction) {
-        fetchContactsController.loadMoreContacts(
-          offset: fetchContactsController.lastContactIndexNotifier.value
-        );
+      if(allowLoadMore) {
```

Comment:
```suggestion
      if (allowLoadMore) {
```

when this variable is changed